### ISSUE TITLE
Charset problems

### ISSUE THREAD

**Issue by odwrotnie** [dmurvihill/courier#10]
I have a problem with encoding. Am I doing something wrong or there is a problem with Coureir? How can I fix this?

I am sending:

```
  <html>
    <head>
      <meta charset="UTF-8" />
    </head>
    <body>
      <h1>Diacritics: żółć</h1>
    </body>
  </html>
```

and getting:

```
<h1>Diacritics: ?�??</h1>
```

the email looks like this:

```
------=_Part_0_1597551722.1405604258142
Content-Type: text/html; charset=UTF-8
Content-Transfer-Encoding: quoted-printable

<html>
    <head>
      <meta charset=3D"text/html; charset=3DUTF-8"/>
    </head>
    <body>
      <h1>Diacritics: ?=F3??</h1>
    </body>
  </html>
------=_Part_0_1597551722.1405604258142--
```

**Comment by rahulsinghai** [comment]
Current version also does not allow to set encoding at the Subject level. In our case we have special characters in subject as well.

**Comment by danielgratzl** [comment]
Subject charset problem has been resolved https://github.com/softprops/courier/pull/29

**Comment by daddykotex** [comment]
It should be fixed in 1.0.0-RC1.
